scripts/redirect_creation/main.tf

```diff
@@ -0,0 +1,52 @@
+
+terraform {
+  backend "remote" {
+    organization = "liquibase"
+    workspaces {
+      name = "liquibase-docs"
+    }
+  }
+}
+
+# use Terraform Remote State Data stanza to read state from prod and staging workspaces
+
+data "terraform_remote_state" "prod" {
```

Comment:
are these two data sources required?

Reply:
mmmm I should yes, instead of that we would need to hardcode their values in the `resource "aws_s3_object" "redirects"`